### PR TITLE
Feature/integration models

### PR DESCRIPTION
**Pull request by jan-schutte**

### CODE REVIEW

File: samples/.nox/design/application/integrations/CountrySync.integration.nox.yaml
Comment (rochar) [inline samples/.nox/design/application/integrations/CountrySync.integration.nox.yaml:1]: @jan-schutte the idea its to merge all this in Nox.Integration project? do we do it on this issue https://github.com/NoxOrg/Nox.Generator/issues/976?

File: src/Nox.Solution/Validation/Application/Integration/Source/IntegrationScheduleValidator.cs
Comment (rochar) [inline src/Nox.Solution/Validation/Application/Integration/Source/IntegrationScheduleValidator.cs:1]: the cron format, should we not validate that this is a valid cron input

File: src/Nox.Integration/Abstractions/INoxIntegrationAdapter.cs
Comment (rochar) [inline src/Nox.Integration/Abstractions/INoxIntegrationAdapter.cs:1]: duplicated in Nox.Integration.Abstractions

File: src/Nox.Integration.Abstractions/Nox.Integration.Abstractions.csproj
Comment (rochar) [inline src/Nox.Integration.Abstractions/Nox.Integration.Abstractions.csproj:1]: if we do not merge into a single project we will need to produce a package also?

File: src/Nox.Integration.Adapter.SqlServer/Nox.Integration.Adapter.SqlServer.csproj
Comment (rochar) [inline src/Nox.Integration.Adapter.SqlServer/Nox.Integration.Adapter.SqlServer.csproj:1]: if we do not merge into a single project we will need to produce a package also?

**Review by rochar — CHANGES_REQUESTED** [review]

**Review by AndreyDegtyarev — DISMISSED** [review]

File: src/Nox.Solution/Validation/Application/Integration/Source/IntegrationScheduleValidator.cs [hunk 1]
```diff
@@ -4,11 +4,11 @@ namespace Nox.Solution.Validation
 {
     internal class IntegrationScheduleValidator: AbstractValidator<IntegrationSchedule>
     {
-        public IntegrationScheduleValidator(string sourceName, string integrationName)
+        public IntegrationScheduleValidator(string integrationName)
         {
             RuleFor(p => p.Start)
                 .NotEmpty()
```
Comment (rochar) [inline src/Nox.Solution/Validation/Application/Integration/Source/IntegrationScheduleValidator.cs]: duplicates Required? Not needed?

**Pull request merged**